config.md

```diff
@@ -244,7 +244,7 @@ This specification uses the following terms:
             Implementations MUST generate an error if they encounter a unknown value while verifying or unpacking an image.
           </li>
           <li>
-            <code>diff_ids</code> is an array of layer content hashes (<code>DiffIDs</code>), in order from bottom-most to top-most.
+            <code>diff_ids</code> is an array of layer content hashes (<code>DiffIDs</code>), in order from bottom-most to top-most and bottom-most at index 0.
```

Comment:
@wking  Frow what I understanding, `layers` and `diff_ids` are different concept at some point. `layers` is defined in manifest to reference layers in image layout, and `diff_ids` is defined in `config.json` to reference layers in image local storage for engine like docker.